presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1132,3 +1132,39 @@ In the following query, the expression CURRENT_TIMESTAMP returns the current tim
             20 | canada        |         2 | comment
             30 | mexico        |         3 | comment
     (3 rows)
+
+Sorted Tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector supports to create sorted files.
+Data in the Iceberg table is sorted during the writing process within each file.
+
+Sorted Iceberg tables can provide a huge increase in performance in query times.
+Sorting is particularly beneficial when the sorted columns show a
+high cardinality and are used as a filter for selective reads.
+
+The sort order is configured with the ``sorted_by`` table property.
```

Comment:
```suggestion
Configure sort order with the ``sorted_by`` table property to specify an array of one or more columns to use for sorting.
```
You may want to break this suggestion into two lines for line length considerations, but this was the easiest way to show my suggested text change. Let me know if this rephrasing suggestion is incorrect, please!